third-party/llvm/Makefile

```diff
@@ -204,6 +204,15 @@ system: $(LLVM_CLANG_CONFIG_FILE)
 system-minimal:
 
 none:
+ifndef CHPL_LLVM
```

Comment:
I haven't tried to test this but I would think you would want to use `CHPL_MAKE_LLVM` here?

Reply:
I actually do want `CHPL_LLVM`. The build system infers `CHPL_MAKE_LLVM` to be "none" in the case where this message should come out. It should get the error when no system LLVM is found and bundled is not built and CHPL_LLVM is unset. It requests that the user explicitly set `CHPL_LLVM` in that case, probably to either "bundled" or "none".